Supplement Map maker's README.md (#552)
MT-CTF/capturetheflag: mods/ctf/ctf_map/map_maker/README.md

* `phys_jump`: [Optional] Player jump multiplier. Accepts any valid number. Defaults to 1.
* `phys_gravity`: [Optional] Player gravity multiplier. Accepts any valid number. Defaults to 1.

#### `license`

* Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:

  ```lua
  license = <name>
  ```

> * If license require something (for example if you modify other's map and you have to tell who is author of the original map) you have to include this in `license`.

Reviewer: Slightly improved wording; hope you don't mind.

```suggestion
* If attribution is required (for example if you modify other's map and you have to tell who is author of the original map), that has to be appended to the `license` field.
```

If you want to tell more infomation, you can use:

  ```lua
  others = <description>
  ```

* If you don't know which license to use, [this list of CC licenses](https://creativecommons.org/use-remix/cc-licenses/) can help you.
* We can only accept Free Software licenses, e.g.`CC BY-SA 4.0`.
* Please know what you are doing when choosing a certain license. For example, you can read information about various licenses and/or consult a lawyer.

#### `treasures`

`treasures` is a list of treasures to be registered for this map in serialized format.

An example `treasures` value that registers steel pick, shotgun, and grenade:

  ```lua
  treasures = default:pick_steel,0.5,5,1,10;shooter:shotgun,0.04,2,1;shooter:grenade,0.08,2,1

>   ```

Reviewer: This code-block isn't a part of a list. Please revert the changes to these lines.

(See [here](../../../other/treasurer/README.md) to understand the magic numbers)

### `screenshot`

Every map must have its own screenshot in map's folder. It should have an aspect ratio of 3:2 (screenshot 600x400px is suggested).

It should be named `screenshot.png`.

### `skybox` [Optional]

Six images which should be in map's folder.

* `skybox_1.png` - up
* `skybox_2.png` - down
* `skybox_3.png` - east
* `skybox_4.png` - west
* `skybox_5.png` - south
* `skybox_6.png` - north

You have to include skybox license in `license` in `.conf` file. We can only accept Free Software licenses, e.g. `CC0`, `CC BY 3.0`, `CC BY 4.0`, `CC BY-SA 3.0`, `CC BY-SA 4.0`.

> (You can find some good skyboxes with suitable licenses at [https://opengameart.org](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)

Reviewer: I think the outer pair of parenthesis isn't required. i.e. `(You can find some ... )` -> `You can find some ... .`

* `phys_jump`: [Optional] Player jump multiplier. Accepts any valid number. Defaults to 1.
* `phys_gravity`: [Optional] Player gravity multiplier. Accepts any valid number. Defaults to 1.

#### `license`

* Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:

> ```lua

Reviewer 2: code blocks in bullet lists should be indented so that they are part of the lists, and also have blank lines before and after

Author: Good? 👇

Reviewer: No, the backticks themselves have to be indented by two spaces, along with the contents of the code block.

It seems like you've also indented the contents of code-blocks with tabs.

Author: What about now?

license = <name>
```
* If license require something (for example if you modify other's map and you have to tell who is author of the original map) you have to include this in `license`.
If you want to tell more infomation, you can use:
```lua
others = <description>
```
* If you don't know which license to use, [this list of CC licenses](https://creativecommons.org/use-remix/cc-licenses/) can help you.

> * Recommended license is `CC BY-SA 4.0`, a very good all-rounder license, which most of our maps use.

Reviewer 3: ```suggestion
* We can only accept Free Software licenses, e.g.`CC BY-SA 4.0`.
```

Reviewer: Even CC BY-NC is a "free" license. It's open-source licenses that we're looking for, not free ones. :)

Reviewer 3: No, CC BY-NC is not a Free license: https://www.gnu.org/licenses/license-list.html#CC-BY-NC

(Please also not the capitalized "F": It implies that not "free" as in _free beer_ is meant, but as in _Free Speech_.

Please read about the difference of Open Source and Free Software: https://www.gnu.org/philosophy/open-source-misses-the-point.en.html

Reviewer: Ah, a capitalised `F` means a completely different thing. Kewl...

> Unless you know what you're doing, we recommend choosing `CC BY-SA 4.0` and don't just use it without having read any information about it.

Reviewer 3: ```suggestion
* Please know what you are doing when choosing a certain license. For example, you can read information about various licenses and/or consult a lawyer.
```

* Every map must have its own license. Once you've chosen your license, simply add the following line to the map's `.conf` file:

  ```lua

> 

Reviewer: These empty lines within the code blocks aren't required

Author: Ok, fixing

Author: fixed

```

(See [here](../../../other/treasurer/README.md) to understand the magic numbers)

### `screenshot`

> Every map must have its own screenshot in map's folder. It should have ratio around 3:2 (screenshot 600x400 is suggested).

Reviewer 3: ```suggestion
Every map must have its own screenshot in map's folder. It should have an aspect ratio of 3:2 (screenshot 600x400px is suggested).
```

Author: :+1:

It should be named `screenshot.png`.

### `skybox` [Optional]

Six images which should be in map's folder.

* `skybox_1.png` - up
* `skybox_2.png` - down
* `skybox_3.png` - east
* `skybox_4.png` - west
* `skybox_5.png` - south
* `skybox_6.png` - north

> (You can find some good skyboxes with suitable licenses [here](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)

Reviewer 3: ```suggestion
(You can find some good skyboxes with suitable licenses at [https://opengameart.org](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)
```

It should be named `screenshot.png`.

### `skybox` [Optional]

Six images which should be in map's folder.

* `skybox_1.png` - up
* `skybox_2.png` - down
* `skybox_3.png` - east
* `skybox_4.png` - west
* `skybox_5.png` - south
* `skybox_6.png` - north

You have to include skybox license in `license` in `.conf` file

(You can find some good skyboxes with suitable licenses at [https://opengameart.org](https://opengameart.org/art-search-advanced?field_art_tags_tid=skybox).)

## Editing exported map

The easiest way to edit exported maps is the following:
* Create a world using `singlenode` mapgen. Enable `WorldEdit` and `ctf_map` mod,
* Go in the world's folder, create a folder named `schems`, and place the `.mts` file inside,
* Start the game, `/grantme all` and enable `fly` (there is no ground in singlenode mapgen),
* Do `//1` to set the position where you will generate the map,
* Do `//mtschemplace yourschematic` (where `yourschematic` is the name of the mts file without `.mts`).

When you finish:

* Place `//1` and `//2` exactly in opposite corners of map (cube),
* Do `//mtschemcreate <new_name>` to create new edited `.mts` file. It will be saved in `schems` folder.

> 

Reviewer: Why so many empty lines? :O

Author: o_O
Fixing...

Author: Done
